src/planner/Admin.h

```diff
@@ -1133,6 +1133,17 @@ class ShowZones final : public SingleInputNode {
         : SingleInputNode(qctx, Kind::kShowZones, input) {}
 };
 
+class ShowStatus final : public SingleInputNode {
```

Comment:
We'd better use statis, not status, is it a typo?

Reply:
Because we use the current state of the space to display the vertices and edges of the space, the number of vertices for each tag, and the number of edges for each edgetype.